hermes/views.py

```diff
@@ -47,6 +51,19 @@ def _extract_hop_auth(request) -> Auth:
     hop_user_auth: Auth = jsons.load(request.session['hop_user_auth_json'], Auth)
     return hop_user_auth
 
+
+def coordinates_valid(data):
+        for row in data:
+            try:
+                ra, dec = float(row['ra']), float(row['dec'])
+                SkyCoord(ra, dec, unit=(units.deg, units.deg))
+            except:
+                try:
+                    SkyCoord(row['ra'], row['dec'], unit=(units.hourangle, units.deg))
+                except:
+                    raise ValidationError('Coordinates do not all have valid RA and Dec')
+
+
 class CandidateDataSchema(Schema):
     candidate_id = fields.String(required=True)
     ra = fields.String(required=True)
```

Comment:
Non-sidereal objects will need orbital elements rather than RA, Dec.  Discovery_date shouldn't be a required field, for some science cases this isn't critical and may not be known.